ts41/get.d.ts

```diff
@@ -0,0 +1,82 @@
+import {Split} from '../source/split';
+import {Integers} from '../source/utilities';
+
+/**
+ * Gets a deeply-nested property from an object, like lodash's `get` method.
+ *
+ * Use-case: retrieve a property from deep inside an API response or other complex object.
+ *
+ * @example
+ * import { Get } from 'type-fest'
+ *
+ * interface ApiResponse {
```

Comment:
This is what [elasticsearch responses](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-body.html) look like. Just as an example of the kind of very-nested payloads you can get in the real world. The `_source` document is a [FHIR Patient](http://www.hl7.org/fhir/patient.html) - again, wanted to keep it somewhat realistic.